### ISSUE TITLE
incorrect mapping for wireless xbox one S controller

### ISSUE THREAD

**Issue by zJanny** [gabomdq/SDL_GameControllerDB#443]
When playing PCSX2 under Linux, some buttons are not mapped or mapped wrong.
For example the Select and Start buttons are mapped to things like throwing items for example.
However when testing the controller in the web, everything works fine, so it seems to be an issue with the SDL implementation of the mappings.

Controller ID is: 050000005e040000e002000003090000

**Comment by cgriffin4** [comment]
I'm seeing the same issue.

**Comment by offalynne** [comment]
If you have a problem with PCSX2, take it to PCSX2. If you want to fix the map, then feel free to issue a pull request.

**Comment by cgriffin4** [comment]
> If you have a problem with PCSX2, take it to PCSX2. If you want to fix the map, then feel free to issue a pull request.

Apologies, my issue isn't with PCSX2. 

I used the mapping file with Love2D engine and the mapping was the same reported mismatched keybindings (left bumper and right bumper are mapped to select/start for instance) when connected via bluetooth. I don't have the knowledge to correct the mappings but if I figure it out I'll put in a pull request. 

Thank you.

**Comment by cgriffin4** [comment]
Yes, it was only when using in Bluetooth mode, I tried using the Gamepad tool but it won't register the trigger buttons which is the same behavior I noticed when using the raw input from my lua script. The solution was to install an OS update KB 4580386 for me but I think that varies depending on which Windows 10 build you are on. After that be sure to remove the controller from bluetooth list and repair.

Sorry for the inconvenience, it was an issue with my OS, not the mapping. Thank you for the help!
